app/javascript/application.js

```diff
@@ -1,3 +1,27 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
+import "@hotwired/turbo-rails";
+import "controllers";
+
+document.addEventListener("DOMContentLoaded", () => {
+  // Get all "navbar-burger" elements
+  const $navbarBurgers = Array.prototype.slice.call(
+    document.querySelectorAll(".navbar-burger"),
+    0
+  );
```

Comment:
I know that you got this from Bulmas website, but I wonder why they used weird stuff live $ for the name of the variables. Or things like this instead of just doing:

```javascript
const $navbarBurgers = [...document.querySelectorAll('.navbar-burger')];
```
or even just

```
const $navbarBurgers = .document.querySelectorAll('.navbar-burger');
```

Reply:
yeah i was wondering that as well. Do you think i should refactor? or leave as is?